doc/src/stdlib/numbers.md

```diff
@@ -2,22 +2,39 @@
 
 ## Standard Numeric Types
 
-  * `Bool`
-  * `Int8`
-  * `UInt8`
-  * `Int16`
-  * `UInt16`
-  * `Int32`
-  * `UInt32`
-  * `Int64`
-  * `UInt64`
-  * `Int128`
-  * `UInt128`
-  * `Float16`
-  * `Float32`
-  * `Float64`
-  * `Complex64`
-  * `Complex128`
```

Comment:
are the Inf32, NaN32, etc constants documented separately? if so then I think these should be mentioned somewhere. in the Complex docstring is better than not mentioning them anywhere